.github/workflows/docker-slurm.yml

```diff
@@ -44,24 +44,15 @@ jobs:
           cat mirrors.in.yaml | envsubst > mirrors.yaml
           ./create_dockerfile.sh
       -
-        name: Make lint
+        name: Make Lint
         run:  |
           pip install flake8
           flake8 src tests
       -
-        name: Build spack-stack flux-cache
-        uses: docker/build-push-action@v4
-        with:
-          context: ./docker/spack-stack
-          file: ./docker/spack-stack/Dockerfile.flux-only
-          secret-files: "mirrors=./docker/spack-stack/mirrors.yaml"
-          secrets: |
-            "spack_stack_buildcache_key=${{ secrets.SPACK_STACK_BUILDCACHE_KEY }}"
-            "spack_stack_buildcache_secret_key=${{ secrets.SPACK_STACK_BUILDCACHE_SECRET_KEY }}"
-          push: true
-          tags: ghcr.io/noaa-gsl/exascaleworkflowsandbox/flux-cache:latest
-          cache-from: type=registry,ref=ghcr.io/noaa-gsl/exascaleworkflowsandbox/flux-cache:cache
-          cache-to: type=registry,ref=ghcr.io/noaa-gsl/exascaleworkflowsandbox/flux-cache:cache,mode=max
```

Comment:
This section needs to stay. It probably got mistakenly removed during your last merge with main.